src/util/processA11yRatings.ts

```diff
@@ -1,14 +1,18 @@
-import { PlaceWithAccesibilityData } from "./placeTypes";
+import { PlaceWithA11yData } from "peer-types";
 
-export const getAverageA11yRating = (place: PlaceWithAccesibilityData): number => {
+export const getAverageA11yRating = (place: PlaceWithA11yData): number => {
 	if (!place.accessibilityData) return -1;
 	// add each of the average field values together
 	const total =
-		Number(place.accessibilityData.avgBraille) +
-		Number(place.accessibilityData.avgFontReadability) +
-		Number(place.accessibilityData.avgGuideDogFriendly) +
-		Number(place.accessibilityData.avgNavigability) +
-		Number(place.accessibilityData.avgStaffHelpfulness);
+		Number(place.accessibilityData.spacingAvg) +
+		Number(place.accessibilityData.guideDogAvg) +
+		Number(place.accessibilityData.lightingAvg) +
+		Number(place.accessibilityData.noiseLevelAvg) +
+		Number(place.accessibilityData.isStaffHelpfulAvg) +
+		Number(place.accessibilityData.isMenuAccessibleAvg) +
+		Number(place.accessibilityData.isStairsRequiredAvg) +
+		Number(place.accessibilityData.isBathroomOnEntranceFloorAvg) +
+		Number(place.accessibilityData.isContactlessPaymentOfferedAvg);
 	const count = 5;
```

Comment:
I
```suggestion
		Number(place.accessibilityData.spacingAvg) +
		Number(place.accessibilityData.guideDogAvg) +
		Number(place.accessibilityData.lightingAvg) +
		Number(place.accessibilityData.noiseLevelAvg) +
		Number(place.accessibilityData.isStaffHelpfulAvg) * 5+
		Number(place.accessibilityData.isMenuAccessibleAvg) * 5+
		Number(place.accessibilityData.isStairsRequiredAvg) * 5+
		Number(place.accessibilityData.isBathroomOnEntranceFloorAvg) * 5+
		Number(place.accessibilityData.isContactlessPaymentOfferedAvg) * 5;
	const count = 9;
```
@aej11a is this the correct suggestion?

Reply:
yes @carpenterd777